src/components/BridgeStatistics.js

```diff
@@ -2,36 +2,30 @@ import React from 'react'
 import numeral from 'numeral'
 import { DataBlock } from './DataBlock'
 
-export const BridgeStatistics = ({ gasValue, users, totalBridged, homeBalance, foreignSupply }) => (
+export const BridgeStatistics = ({ gasValue, users, totalBridged, homeBalance, foreignSupply, homeSymbol, foreignSymbol }) => (
   <div className="statistics-bridge-data">
-    {/*<DataBlock
-      description="Gas Saved"
-      value={numeral(gasValue).format('0.0 a')}
-      type=''
-    />
-    <div className="separator" />*/}
     <DataBlock
       description="Users"
       value={numeral(users).format('0,0')}
       type=''
     />
     <div className="separator" />
     <DataBlock
-      description="Total POA Bridged"
+      description={`Total ${homeSymbol} Bridged`}
       value={numeral(totalBridged).format('0,0.00 a', Math.floor)}
-      type='POA'
+      type={homeSymbol}
     />
     <div className="separator" />
     <DataBlock
-      description="Locked POA in Bridge Contract"
+      description={`Locked ${homeSymbol} in Bridge Contract`}
```

Comment:
Please _grep_ other source files, they could refer to `POA20` as well. E.g. I found this name of the token here:

https://github.com/poanetwork/bridge-ui/blob/8dab5d9ee356e63b3ef324ebcbbdc2b0b516d640/src/components/DailyQuotaModal.js#L36-L37

Reply:
Thanks, fixed